### DISCUSSION TITLE
mfs vs pfrd

### DISCUSSION THREAD

**Discussion by tigerkzr** [trapexit/mergerfs#1080] [Q&A] [answered]
Regarding mfs and pfrd,  my initial thought is that mfs would be good for a pool of similar sized disks and pfrd would be better for a mixed size pool. 

If I am interpreting the prfd policy correctly, I kind of like the idea of drives in a mixed size pool maintaining a similar fill percentage so that they see similar usage(reads,writes,etc).  It seems like, in a mixed size pool, mfs would be a kind of cascade fill from the larger disks to the smaller disks (eg a 2TB disk would see no use until the 16tb is at down to 2TB). Is my interpretation about right, or is there something I am missing? 

Just for reference: the pool I am planning will have 16TBx2 8TBx1, 2TBx2 disks and I had planned to balance the drives once configured.

Thanks in advance

**Comment by trapexit** [comment, marked as answer]
Depends on what you mean by "better".

If you want to actively use the drives and they are empty then yes... mfs would "over use" the largest filesystem. pfrd would spread them out based on available space (statistically).

**Comment by tigerkzr** [comment]
that's more or less what I suspected. Thanks for the answer!
